app/javascript/packs/ordersForm.js

```diff
@@ -0,0 +1,20 @@
+import flatpickr from "flatpickr";
+
+$(document).ready(function() {
+  
```

Comment:
we usually don't leave this blank space here

Reply:
Great, thanks! Commit: b5b13dd